### PR TITLE
Add overridable on "ProductAttributeValue" and "ProductAttribute"

### PR DESCRIPTION
**Pull request by yazanhorani**

### CODE REVIEW

**Comment by yazanhorani** [comment]
My life will be much easier if this merged :smile:

**Comment by yazanhorani** [comment]
I can not check the Travis CI build... what shall I do now?

**Comment by specialunderwear** [comment]
run the tests yourself first?

**Comment by yazanhorani** [comment]
Forgive my ignorance

File: oscarapi/serializers/product.py [hunk 1]
```diff
@@ -47,15 +47,15 @@ class ProductAttributeValueSerializer(OscarModelSerializer):
 
     class Meta:
         model = ProductAttributeValue
-        fields = ('name', 'value',)
+        fields = overridable('OSCARAPI_PRODUCT_ATTRIBUTE_VALUE_FIELDS', default=(('name', 'value',)))
```
Comment (specialunderwear) [inline oscarapi/serializers/product.py]: why do you write:
```
default=(('name', 'value',))
```
why so many brackets

File: oscarapi/serializers/product.py [hunk 2]
```diff
@@ -47,15 +47,15 @@ class ProductAttributeValueSerializer(OscarModelSerializer):
...
 
 
 class ProductAttributeSerializer(OscarModelSerializer):
     productattributevalue_set = ProductAttributeValueSerializer(many=True)
 
     class Meta:
         model = ProductAttribute
-        fields = ('name', 'productattributevalue_set')
+        fields = overridable('OSCARAPI_PRODUCT_ATTRIBUTE_FIELDS', default=(('name', 'productattributevalue_set')))
```
Comment (specialunderwear) [inline oscarapi/serializers/product.py]: same here

**Comment by SalahAdDin** [comment]
👍

**Comment by yazanhorani** [comment]
why the last test got cancelled?

**Pull request merged**